### DISCUSSION TITLE
Polygon in the Map component

### DISCUSSION THREAD

**Discussion by tomtomkratos** [sqlpage/SQLPage#105] [Q&A]
Hello,
First of all, thank you for your work.
SQLpage is amazingly efficient.
Here is my question:
Concerning the map component, is it possible to visualize polygonal geometry? 
That would be interesting, because in Spatialite, you can have points, lines or polygons. 
Thank you very much for your reply.

**Comment by lovasoa** [comment]
Hello and welcome to SQLPage !

  Updates to the map component are currently not high on my priority list, but I might come them later.

  If you need this feature for the next release, and are ready to fund its development, please drop me a line on `contact@ophir.dev`

Cheers,
Ophir

**Comment by tomtomkratos** [comment]
Thank you for your reply and for your work.
This update would be interesting, but of course I can wait. There's no rush.
All the best. 
Tom

**Comment by lovasoa** [comment]
Closing this since this is now supported. https://sql.ophir.dev/documentation.sql?component=map#component
